buildprocess/webpack-tools.config.js

```diff
@@ -28,7 +28,6 @@ module.exports = function () {
     plugins: [
       new MiniCssExtractPlugin({
         filename: "TerriaJS.css",
```

Comment:
The 1.3.0 release adds checking of unknown options. I cannot find anything about the `disable` option in the release notes of mini-css-extract-plugin for any version, so I assume it is some leftover from elsewhere that should be removed.